ecosystem/sep-0010.md

```diff
@@ -300,6 +307,12 @@ This setup allows the **Server** to verify that the challenge returned by the **
 
 **Servers** may chose which **Client Domains** to verify. If the **Client** requests verification of its domain but the **Server** has no use for verifying that domain, the **Server** should proceed as if the **Client** did not provide the domain in the request for the challenge transaction. If the **Server** attempts but is unable to fetch the `SIGNING_KEY` from the provided **Client Domain**, the **Server** should return a `400 Bad Request` HTTP status code.
 
+### Memos
+
+Stellar transaction memos are used for a variety of purposes in the ecosystem, but in the context of this standard memos attached to challenge transactions are typically used to distinguish users who share a single Stellar account. This is especially relevant for applications who custody user funds in shared Stellar accounts.
```

Comment:
❗ A couple suggestions here, that I feel a bit stronger about, but let me know your thoughts.

- We should be as clear as possible, even if verbose, about the separation between sessions that use memos.
- Mentioning custody seems unnecessary here.

```suggestion
Stellar transaction memos are used for a variety of purposes in the ecosystem, but in the context of this standard memos attached to challenge transactions distinguish sessions that should be entirely separate, scoped, and detached from each other. Two users who are authenticated with the same Stellar account but different memos should have the same level of separation as two users who are authenticated with different Stellar accounts. Typically the sessions are unique users who share a single Stellar account.
```